src/contract/method.rs

```diff
@@ -0,0 +1,214 @@
+//! Implementation for a contract method builder and call future. This is not
```

Comment:
If you use git mv it should be able to track and only highlight the changes between the files instead of saying all lines are added.

Reply:
:open_mouth: